src/commands/tsgen.ts

```diff
@@ -91,38 +104,61 @@ export default class TypeScriptCodeGeneratorCommand extends Command {
         this.error('Please provide an output path.', {exit: 2})
       }
 
+      let region = ''
+      switch (this.region.name) {
+      case 'NA':
+        region = 'US'
+        break
+      case 'AZURE-NA':
+        region = 'AZURE_NA'
+        break
+      case 'AZURE-EU':
+        region = 'AZURE_EU'
+        break
+      case 'GCP-NA':
+        region = 'GCP_NA'
+        break
+      default:
+        region = this.region.name.toUpperCase()
+      }
+
```

Comment:
@praveen-mohan-cs , @meetstack19 How can type creation be tested in a development or staging environment? Here, it is hardcoded.
 https://github.com/contentstack/types-generator/blob/main/src/generateTS/index.ts#L12. 
 
 CC: @netrajpatel

Reply:
removed